### ISSUE TITLE
No "web server" listed as instructions say to expect

### ISSUE THREAD

**Issue by j-vasil** [FirebaseExtended/codelab-friendlyeats-flutter#14]
In step 4 (Prepare the Firebase and Flutter CLI) it says: "In your IDE under the devices pulldown, or at the command line using flutter devices, you should now see Chrome and Web server listed."  I see Chrome and "Edge" but no "Web server".

PS C:\...\friendlyeats-flutter> flutter devices
Downloading Material fonts...                                      796ms
Downloading Gradle Wrapper...                                       47ms
Downloading package sky_engine...                                  363ms
Downloading flutter_patched_sdk tools...                         1,176ms
Downloading flutter_patched_sdk_product tools...                   797ms
Downloading windows-x64 tools...                                 1,600ms
Downloading windows-x64/font-subset tools...                       387ms
2 connected devices:

Chrome (web) • chrome • web-javascript • Google Chrome 89.0.4389.114
Edge (web)   • edge   • web-javascript • Microsoft Edge 89.0.774.68

**Comment by ditman** [comment]
This is correct, the web-server device was hidden recently (to be used mainly for testing purposes).

For the web-server device to show up, and be usable, the flutter command needs to be called with the `--show-web-server-device` flag, like so:

```
$ flutter --show-web-server-device devices
2 connected devices:

Web Server (web) • web-server • web-javascript • Flutter Tools
Chrome (web)     • chrome     • web-javascript • Google Chrome 90.0.4430.70 beta
```
